packages/edit-site/src/components/global-styles/color-palette-panel.js

```diff
@@ -89,6 +96,10 @@ export default function ColorPalettePanel( { name } ) {
 				slugPrefix="custom-"
 				popoverProps={ popoverProps }
 			/>
+			{ /* @TODO: pass down variations to component? */ }
+			{ !! colorVariations.length && (
+				<ColorVariations variations={ colorVariations } />
```

Comment:
I'm playing around and wondering... for colors, selecting a variation merges all color values into the current user styles and therefore the editor canvas: settings, styles, elements, blocks and so on. 

It doesn't just replace current presets/palettes.

Since it's under "Palettes" at the moment, should it just replace color settings, and therefore presets?

This would be consistent behaviour, for this panel at least.

Otherwise, if we're merging all a variation's  colors to the current user styles, maybe this component be moved from under "palettes" up a level to the colors, and renamed... to, I don't know, something like "Apply a color variation" or something.

Reply:
I think it's fine to leave the presets in Colors > Palette, as they are presets for the palette first (which consequently change the other values, as expected). Otherwise there's too much of a disconnect between the presets and the palette. 

I'd consider moving it above "Custom" even, as those custom colors are appended to the palette. 

![CleanShot 2024-02-16 at 11 28 47](https://github.com/WordPress/gutenberg/assets/1813435/baf0541d-a9ee-426f-b08c-4abf27a25f21)